src/get-subseqs.py

```diff
@@ -18,8 +18,31 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
-    print(f"and the coordinates in {args.coords}")
+    # Get things from cords file
+    text = ""
+    with open(args.fasta.name + "-coordinates", "r") as file:
+        text = file.read()
+    
+    coords = text.split("\n")
+    # extract from fasta file
+
+    with open(args.fasta.name, "r") as file:
+        text = file.read()
+        chrom = text.split(">")
+    
+    genomeDict = dict()
+    for i in range(1,len(chrom)):
+        name, seq = chrom[i].split("\n", 1)
+        name = name.strip()
+        seq = seq.replace("\n", "")
+        genomeDict.update({name : seq})
```

Comment:
`genomeDict[name] = seq` would be the conventional way to add a key-value mapping to a dictionary.